packages/card/src/react/__stories__/index.story.tsx

```diff
@@ -18,25 +19,24 @@ const longStringsMetaData = [
 const longTitle =
   'The First Course of Our Age to Teach the Minute Waltz to an Eight Horn Fugle Hummer in Less Than the Average of the First 60 Seconds'
 
-const CardWithDefaults = props => <Card {...props} />
-
-CardWithDefaults.defaultProps = {
-  title: <Card.Title>Card Title</Card.Title>,
-  image: <Card.Image src={getImgSrc()} />
+const CardWithDefaults = (props: Partial<CardProps>) => {
+  const title = props.title || <Card.Title>Card Title</Card.Title>
+  const image = props.image || <Card.Image src={getImgSrc()} />
+  return <Card {...props} title={title} image={image} />
 }
 
 const sizes = {
   [Card.sizes.small]: { width: '140px' },
   [Card.sizes.medium]: { width: '320px' },
   [Card.sizes.large]: { width: '540px' }
 }
-const ConstrainSizeDecorator = size => storyFn => (
-  <div style={sizes[size]}>{storyFn()}</div>
-)
+const ConstrainSizeDecorator = (size: keyof typeof sizes) => (
```

Comment:
```suggestion
const ConstrainSizeDecorator = (size: ValueOf<typeof sizes>) => (
```

i've been checking the value instead of the key. that has made the most sense to me

Reply:
Yes, I think that is the common use case.  In this case, you can see on line 35 that `size` is actually a key in this case.